exercises/hamming/hamming_test.go

```diff
@@ -2,26 +2,40 @@ package hamming
 
 import "testing"
 
-const targetTestVersion = 4
+const targetTestVersion = 5
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
```

Comment:
if this will be added, shall I suggest it get removed from `TestHamming`? We will only need to check it in one place.